server/sonar-server/src/main/java/org/sonar/server/permission/index/PermissionIndexerDao.java

```diff
@@ -83,18 +89,20 @@ public Dto addGroup(String s) {
     "  project_authorization.project as project, " +
     "  project_authorization.user_id as user_id, " +
     "  project_authorization.permission_group as permission_group, " +
-    "  project_authorization.updated_at as updated_at " +
+    "  project_authorization.updated_at as updated_at, " +
+    "  project_authorization.qualifier as qualifier " +
     "FROM ( " +
 
     // project is returned when no authorization
     "      SELECT " +
     "      projects.uuid AS project, " +
     "      projects.authorization_updated_at AS updated_at, " +
+    "      projects.qualifier AS qualifier, " +
     "      NULL AS user_id, " +
     "      NULL  AS permission_group " +
     "      FROM projects " +
     "      WHERE " +
-    "        projects.qualifier = 'TRK' " +
+    "        (projects.qualifier = 'TRK' or  projects.qualifier = 'VW') " +
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "        (projects.qualifier = 'TRK' or  projects.qualifier = 'VW') " 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)